packages/rx_bloc_cli/mason_templates/bricks/feature_otp/README.md

```diff
@@ -0,0 +1,130 @@
+# feature_otp
```

Comment:
In the other brick folders except for the lib_social_logins there is no specific information for the brick, but instead some general information and links, which is the same in every brick folder, only the first line is different, where the name of the feature is specified for example in brick/feature_counter/README.md the name is:
`# feature_counter`, so I think that this file content should be replaced to be as in the other bricks

Reply:
I think we need to add more detailed readme for the bricks, other ones basically don't have readme its just generated one